libraries/CurieBLE/src/BLEPeripheral.cpp

```diff
@@ -21,7 +21,7 @@
 
 #include "BLEPeripheral.h"
 
-static BLEPeripheralEventHandler m_eventHandlers[BLEDeviceLastEvent];
+static BLEPeripheralEventHandler m_eventHandlers[BLEDeviceLastEvent] = {NULL, NULL, NULL, NULL};
```

Comment:
This is unncessary-- `m_evenHandlers` is declared static, so it will be located in .bss
All elements will be initialised to 0 by default.